pkgs/development/libraries/yder/default.nix

```diff
@@ -0,0 +1,47 @@
+{ stdenv, fetchFromGitHub, cmake, orcania, systemd, check, subunit }:
+stdenv.mkDerivation rec {
+  pname = "yder";
+  version = "1.4.6";
+
+  src = fetchFromGitHub {
+    owner = "babelouest";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "0j46v93vn130gjcr704rkdiibbk3ampzsqb6xdcrn4x115gwyf5i";
+  };
+
+  patches = [
+    # We set CMAKE_INSTALL_LIBDIR to the absolute path in $out, so
+    # prefix and exec_prefix cannot be $out, too
+    ./fix-pkgconfig.patch
+  ];
+
+  nativeBuildInputs = [
+    cmake
+  ];
+
+  buildInputs = [
+    orcania systemd
+  ];
+
+  checkInputs = [
+    check subunit
+  ];
+
+  cmakeFlags = [
+    "-DBUILD_YDER_TESTING=on"
+  ];
+
+  doCheck = true;
+
+  preCheck = ''
+    export LD_LIBRARY_PATH="$(pwd):$LD_LIBRARY_PATH"
+  '';
+
+  meta = {
+    description = "Logging library for C applications";
+    homepage = "https://github.com/babelouest/yder";
+    license = stdenv.lib.licenses.lgpl21;
+    maintainers = with stdenv.lib.maintainers; [ johnazoidberg ];
```

Comment:
Since the packages relies on `systemd` it should set `platforms` to `linux` only.